### ISSUE TITLE
transmuted value is not an instance of the annotation after subclass?

### ISSUE THREAD

**Issue by xbanke** [seandstewart/typical#58]
* typical version: 2.0.2
* Python version: 3.7.4
* Operating System:

### Description
@seandstewart , is this a bug?

```py
import typic

@typic.klass
class Foo:
    foo: str


@typic.klass
class FooBar(Foo):
    bar: str


isinstance(Foo.transmute({'foo': 'foo'}), Foo)  # True
isinstance(FooBar.transmute({'foo': 'foo', 'bar': 'bar'}), FooBar)  # False
```

**Comment by seandstewart** [comment]
It most definitely is. This was a tricky bug - but as a result I've got a fix which also cleans up the resulting MRO as well. I'll publish a release with a fix later today.

**Comment by seandstewart** [comment]
This is fixed in v2.0.3, which has been published to PyPI.

**Comment by xbanke** [comment]
Thanks for quick fixing.
